ingestion/src/metadata/ingestion/source/migrate.py

```diff
@@ -0,0 +1,132 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+"""Metadata source module"""
+
+import logging
+
+from metadata.generated.schema.entity.policies.policy import Policy
+from metadata.generated.schema.entity.services.connections.metadata.openMetadataConnection import (
+    OpenMetadataConnection,
+)
+from metadata.generated.schema.metadataIngestion.workflow import (
+    Source as WorkflowSource,
+)
+from metadata.ingestion.api.source import InvalidSourceException, SourceStatus
+from metadata.ingestion.ometa.ometa_api import OpenMetadata
+from metadata.ingestion.source.metadata import MetadataSource
+
+logger = logging.getLogger(__name__)
+
+
+class Policy:
+
+    policy_dict: dict
+
+    def __init__(self, policy_dict) -> None:
+        self.policy_dict = policy_dict
+
+
+class Tag:
```

Comment:
Mixed feelings when shadowing JSON Schema classes

Reply:
Also, why do we have a `migrate` source? We already have an `OpenMetadata` source we could use